### PR TITLE
Use latest urllib3 without secure extra

### PR DESCRIPTION
**Pull request by GSVarsha**
Signed-off-by: Varsha GS varsha.gs@ibm.com

- Use urllib3>=1.26.18
- Remove secure extra feature ([more info](https://github.com/urllib3/urllib3/issues/2680))

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
                         'protobuf<5.0.0',
                         'requests>=2.6.0',
                         'six>=1.12.0',
-                        'urllib3<1.27,>=1.26.5',],
+                        'urllib3>=1.26.18',],
```
Comment (Ferenc-) [inline setup.py:51]: The way I see there is no reason why customers who had to stick with `urllib3`  version `1.26.5`, should not be able to use our tracer without any issue. So we perhaps we could keep this on `>=1.26.5`?

Comment (GSVarsha) [reply setup.py]: Makes sense 👍🏼

**Review by pvital — APPROVED** [review]
It looks good to me.

**Review by Ferenc- — APPROVED** [review]
Approved

**Pull request merged**